### PR TITLE
Fixed formatting of URLs for notebooks and jobs

### PR DESCRIPTION
**Pull request by alexott**
also: 
* added `url` field for clusters & groups.
* small cosmetic fixes to make linter happy

### CODE REVIEW

File: common/azure_auth.go [hunk 1]
```diff
@@ -48,11 +48,13 @@ type AzureAuth struct {
 	temporaryPat            *TokenResponse
 }
 
+// TokenRequest contains request
```
Comment (nfx) [inline common/azure_auth.go:51]: i made those changes in table ACL pr already

**Review by nfx — APPROVED** [review]

**Comment by nfx** [comment]
nice, i like it!

**Comment by nfx** [comment]
@alexott do you want to add a unit test on `FormatURL`?.. codecov complains...

**Comment by alexott** [comment]
test added, I thought that it will be covered by existing tests...

**Comment by nfx** [comment]
@alexott unfortunately golang counts coverage on per-package tests, not on per-repository :/

**Pull request merged**